app/src/main/java/org/oppia/android/app/player/exploration/RequestVoiceOverIconSpotlightListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.app.player.exploration
+
+/** Interface to request voice-over icon spotlight. */
+interface RequestVoiceOverIconSpotlightListener {
+  /** Requests the spotlight fragment to show voice-over icon spotlight. */
```

Comment:
```suggestion
  /** Requests to show a voice-over icon spotlight. */
```

(The specifics of _how_ this happens is an implementation detail, not a contractual requirement).

Reply:
done